apps/internal/migrations/0011_auto_generated_sql.py

```
@@ -0,0 +1,14 @@
+
+
+class Migration(migrations.Migration):
+    dependencies = [('workspaces', '0051_alter_featureconfig_fyle_webhook_sync_enabled'),('internal', '0010_auto_generated_sql')]
+
+    operations = [
+        migrations.RunSQL(
+            sql="""
+                UPDATE feature_configs SET fyle_webhook_sync_enabled = TRUE WHERE fyle_webhook_sync_enabled = FALSE;
+            """
+        )
+    ]
```

Ruff (0.14.4)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)